### PR TITLE
Add Reolink media browser docs

### PR DESCRIPTION
**Pull request by starkillerOG**
## Proposed change

Add Reolink media browser docs

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/103407
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/reolink.markdown [hunk 1]
```diff
@@ -184,6 +184,12 @@ This does the same as pressing the "Check for latest version" in the Reolink app
 Unfortunately this does not always shows the latest available firmware (also not in the Reolink applications).
 The latest firmware can be downloaded from the [Reolink download center](https://reolink.com/download-center/) and uploaded to the camera/NVR manually.
 
+## Media browser for playback of recordings
+
+Depending on the support of the camera, the Reolink integration will provide a media browser through which recorded videos of the camera can be accessed.
+In the sidebar select "Media" -> "Reolink", select the **camera** you want to see recordings of, optionally select if you want a high or low **resolution** stream, select the recording **date**, here all available video files of that day will be shown.
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
In the sidebar, select "Media" > "Reolink" and select the **camera** of which you want to see recordings. Optionally, select if you want a high or low **resolution** stream and select the recording **date**. Here, all available video files of that day will be shown.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @starkillerOG :+1:

**Comment by starkillerOG** [comment]
Thanks for the tweaks :)

**Comment by c0ffeeca7** [comment]
> Thanks for the tweaks :)

Thanks for the integration :-) 
I'm using this review as an excuse to try out a Reolink camera. Love the link to the troubleshooting docs on the firmware update dialog. It helped. 
On another note: so, we're trying to use bold to markup UI elements in the docs. I did not change those in this PR, because " are used consistently here. I guess this would be out of scope for this PR.

**Comment by starkillerOG** [comment]
Yea I will make a separte docs PR to get rid of the ""

**Comment by starkillerOG** [comment]
@c0ffeeca7 the PR for the bold markup is here: https://github.com/home-assistant/home-assistant.io/pull/29951

**Pull request merged**